green_cli/liquid.py

```diff
@@ -27,7 +29,7 @@
 
 # Restrict networks to liquid networks and default to localtest-liquid
 params = {p.name: p for p in green.params}
-params['network'].type = click.Choice(['liquid', 'testnet-liquid', 'localtest-liquid'])
+params['network'].type = click.Choice(reversed([n for n in gdk.get_networks() if 'liquid' in n]))
```

Comment:
Do we want to add single-sig Liquid Mainnet as well?

Reply:
Good point, I've disabled it until our internal acceptance testing is completed.